### ISSUE TITLE
menu/browser font size is inconsitent with zoom

### ISSUE THREAD

**Issue by gaydenko** [surge-synthesizer/surge#6642]
1.1 is running under Linux (Manjaro), KDE. 4K monitor is in use with 200% scaling. The app is used with 300% zooming in. Menu/presets browser font size are to tiny to me readable. Please, look at the attached image. Terminal window on the background is also shown for the reference. A window title also doesn't respect zoom factor.
![SurgeXT-font-size-01](https://user-images.githubusercontent.com/3740016/194405706-5f90c9ca-3ba9-445e-924d-cad02fcd462a.png)

**Comment by baconpaul** [comment]
Yeah the problem here is there is no scale setup api in the standalone so we don't get told (and there in the VST3 spec which Reaper reports to us via juce so it does). It might be an easy fix / omission but it also might not be.

Appreciate the detailed report. Thank you! I'll tag this into the 1.1.2 milestone to make sure that we at least take a look before our next release.

**Comment by mkruselj** [comment]
Possibly related to #5541 and/or #5790 somehow?

**Comment by gaydenko** [comment]
A little addition. While menu scaling is Ok when the app is used as plugin inside a DAW, a dialog on saving a parch with the same name (saying the patch will be overridden) is also very tiny.

**Comment by Brier1104** [comment]
This problem also exists as a CLAP vst in Bitwig Studio.  It's almost impossible to read menus on a 2k monitor with scaling at 300%.

**Comment by baconpaul** [comment]
discord user wandlewolf tells us this impacts dialog boxes messages also like 'rename or delete chicken box'

**Comment by baconpaul** [comment]
With 1.3 those dialogs are all internal so this comes back to 

- clap scaling in cje doesn’t work (which is a separate cje issue) and
- Standalone doesn’t propagate scale through the set scale api properly on Linux

**Comment by baconpaul** [comment]
So the only fix I see to the standalone side of this is to query the dpi on linux and then adjust the juce rendering of the menus with a transform explicitly.

The first seems hard [https://stackoverflow.com/questions/71120399/how-get-correct-dpi-information-on-linux] and the second is really difficult. This doesn't occur in daws with the correct scale factor except in the CLAP, which is a separate issue, so I'm going to bounce this out of the 1.3 milestone. I don't have the particular expertise about scaled linux displays to do it.
